pre_commit_hooks/loaderon_hooks/general_hooks/check_branch_name.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import re
+
+from pre_commit_hooks.loaderon_hooks.util.git_helpers import get_current_branch_name
+from pre_commit_hooks.loaderon_hooks.util.template_methods.checker_template_method import CheckerTemplateMethod
+
+
+class BranchNameChecker(CheckerTemplateMethod):
+    def _add_arguments_to_parser(self):
+        super(BranchNameChecker, self)._add_arguments_to_parser()
+        self.parser.add_argument('-r', '--regex', help='Regex that git current branch must match.')
+
+    def _perform_checks(self):
+        super(BranchNameChecker, self)._add_arguments_to_parser()
```

Comment:
this is a bit weird, usually you don't call `super` on an unrelated method -- maybe a mistake?